### PR TITLE
feat: Add akamai platform

### PR DESCRIPTION
**Pull request by eljohnson92**
# Pull Request

## What? (description)
This PR adds support for the Akamai(Linode) platform
## Acceptance

Please use the following checklist:

- [x] you linked an issue (if applicable)
- [x] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [x] you generated documentation (`make docs`)
- [x] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

**Review by smira — COMMENTED** [review]
if you have some issues you need to resolve before this PR can move forward, please let us know, as we are going to cut 1.7.0-beta on Thursday

File: .drone.jsonnet [hunk 1]
```diff
@@ -1054,6 +1054,8 @@ local release = {
     draft: true,
     note: '_out/RELEASE_NOTES.md',
     files: [
+      '_out/akamai-amd64.raw.gz',
```
Comment (frezbo) [inline .drone.jsonnet:1057]: @smira I guess we can drop these and point to image factory?

**Review by smira — APPROVED** [review]

**Pull request merged**